app/pb/api.proto

```diff
@@ -277,9 +299,8 @@ message UpdateProfileReq {
   HostingStatus hosting_status = 8;
   MeetupStatus meetup_status = 105;
 
-  RepeatedStringValue languages = 9;
-  RepeatedStringValue countries_visited = 10;
-  RepeatedStringValue countries_lived = 11;
+  RepeatedStringValue regions_visited = 30;
+  RepeatedStringValue regions_lived = 31;
```

Comment:
The "exists" in RepeatedStringValue should go away, and be just like RepeatedLanguageAbilityValue.

If you don't want to change RepeatedStringValue right now, make another message without `exists` and use that.